### DISCUSSION TITLE
Warming Up an IP Address on 100% Autopilot

### DISCUSSION THREAD

**Discussion by sals9** [postalserver/postal#1490] [Feature suggestions]
As we know, warming up the IP is a must when we need to target inboxes with a scale - even when we use a so popular SMTP service like Amazon SES. Now, planning the **"gradually increasing"** limit per hour or per day for proper warming up is an easy task. But, applying this plan actually on a server is still a boring manual work with a lot of hassle if we need to reach something like 500k emails per day or 1M emails per day. Because it would take up to 3 months!

It would be a life saving to make a built-in OR addon feature for this useful and important task. And I think something like Postal should has it. Period.

This free tool generates the right plan we should use based on our goal to reach in a safe period of time:
https://h-educate.com/email-marketing-toolkit/

**I hope you can make something similar that we can actually implement it on the server with no hassle. So it got our back and warm up the IP on 100% autopilot.**

**Comment by willpower232** [comment]
#833 is supposedly getting some effort in the near future so might be the answer?
